scripts/src/main/resources/scripts/functions

```diff
@@ -351,6 +351,9 @@ function doExtract() {
   then
     doRunCommand "unzip -q '${1}' -d '${target_dir}'"
   elif [ "${ext}" = ".tar" ]
+  then
+    doRunCommand "tar xf '${1}' -C '${target_dir}'"
+  elif [ "${ext}" = ".xz" ]
   then
     doRunCommand "tar xf '${1}' -C '${target_dir}'"
```

Comment:
shouldnt it be "tar xJf ..." for *.xz files?
BTW: Which software tool is using this odd format?

Reply:
Will that be supported in git-bash (mingw) after all?

Reply:
tar detects the file format, but you are right, maybe adding the option J would be better

Reply:
I"ll check the bash-git tomorrow

Reply:
> Will that be supported in git-bash (mingw) after all?

As you can see in the linked issue (#169) nodejs is provided as .[xz](https://en.wikipedia.org/wiki/Xz) file (`node.js-v10.16.0-linux.tar.xz`) I can see more and more often this file format.

Reply:
> 
> 
> I"ll check the bash-git tomorrow

Ye, it works under git-bash